cirq/optimizers/eject_phased_paulis.py

```diff
@@ -25,10 +25,13 @@
     from typing import Dict, List
 
 
+TVal = Union[float, sympy.Basic]
```

Comment:
we use this type quite a bit, consider moving it somewhere common, maybe cirq.value

Reply:
I don't want to do that in this PR.

Reply:
SG